dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/icon/JdbcIconStore.java

```diff
@@ -98,7 +98,10 @@ public Icon getIconByKey(String key) {
   @Override
   public void save(Icon icon) throws SQLException {
     String sql =
-        "INSERT INTO icon (iconkey,description,keywords,fileresourceid,createdby,created,lastupdated,custom) VALUES (:key,:description,cast(:keywords as jsonb),:fileresourceid,:createdby,now(),now(),true) ";
+        """
+            INSERT INTO icon (iconkey,description,keywords,fileresourceid,createdby,created,lastupdated,custom)
+            VALUES (:key,:description,cast(:keywords as jsonb),:fileresourceid,:createdby,now(),now(),true)
```

Comment:
I commented on this somewhere, why are you using `now()`? We are setting created/lastUpdated by `setAutoFields` in the service.

Reply:
I think yes and I did reply that. 
using `now()` is convenient and does not pose any problem

Reply:
It is odd to do that no? Why are we setting it in Java while we are not even using it since we override it in the SQL. @jbee are we doing this elsewhere? I imagine this can lead to some headaches as its surprising.

Reply:
What sort of headache?

Reply:
We are using this in a few places when we are not using hibernate managed insert and updates. There should not be an issue doing this. You can get odd data when DB "now" is set to another timezone than the DHIS2 server but if that is the case we have a lot of issues :confused: